apps/gateway/src/routes/api/v3/conversations/annotate/annotate.handler.ts

```diff
@@ -1,42 +1,56 @@
 import { AppRouteHandler } from '$/openApi/types'
 import {
   CommitsRepository,
-  DocumentLogsRepository,
+  DocumentVersionsRepository,
   EvaluationsV2Repository,
+  SpanMetadatasRepository,
+  SpansRepository,
 } from '@latitude-data/core/repositories'
 import { AnnotateRoute } from './annotate.route'
 import { annotateEvaluationV2 } from '@latitude-data/core/services/evaluationsV2/annotate'
-import serializeProviderLog from '@latitude-data/core/services/providerLogs/serialize'
-import { findDocumentFromLog } from '@latitude-data/core/data-access/documentLogs'
-import { findLastProviderLogFromDocumentLogUuid } from '@latitude-data/core/data-access/providerLogs'
 import { findProjectFromDocument } from '@latitude-data/core/data-access/projects'
 import { NotFoundError } from '@latitude-data/constants/errors'
 import { serializeEvaluationResultV2 } from './serializeEvaluationResultV2'
-import { HEAD_COMMIT } from '@latitude-data/core/constants'
+import {
+  HEAD_COMMIT,
+  PromptSpanMetadata,
+  SpanType,
+  SpanWithDetails,
+} from '@latitude-data/core/constants'
 
 // @ts-expect-error: broken types
 export const annotateHandler: AppRouteHandler<AnnotateRoute> = async (c) => {
-  const { score, metadata, versionUuid = HEAD_COMMIT } = c.req.valid('json')
+  const {
+    score,
+    metadata: resultMetadata,
+    versionUuid = HEAD_COMMIT,
+  } = c.req.valid('json')
   const { conversationUuid, evaluationUuid } = c.req.valid('param')
   const workspace = c.get('workspace')
   const evaluationsRepo = new EvaluationsV2Repository(workspace.id)
-  const providerLogsRepo = new DocumentLogsRepository(workspace.id)
```

Comment:
What happen when they annotate old evals with provider logs?

Reply:
they won't be able to the ui for logs will not show up anymore